sdk/src/main/java/ch/admin/bag/covidcertificate/sdk/android/data/AcceptedTestProvider.kt

```diff
@@ -40,25 +40,12 @@ class AcceptedTestProvider private constructor(context: Context) {
 		return false
 	}
 
-	fun getTestName(testEntry: TestEntry): String? {
+	fun getManufacturesAndNameLabelIfExists(testEntry: TestEntry): String? {
 		if (testEntry.type == TestType.PCR.code) {
 			return testEntry.naaTestName ?: "PCR"
 		} else if (testEntry.type == TestType.RAT.code) {
-			return testEntry.naaTestName
+			return metadataStorage.productsMetadata.test.manf.valueSetValues[testEntry.ratTestNameAndManufacturer]?.display
```

Comment:
Shouldn't we at least fall back to the raw value (i.e. "1607") if we cannot lookup the test in the value set?